model/auth.go

```diff
@@ -105,9 +106,12 @@ func (a *AuthDefinitions) UnmarshalJSON(b []byte) error {
 		return a.unmarshalFile(b)
 	case '[':
 		return a.unmarshalMany(b)
+	case '{': // single values are not supported, should we support it?
+		return fmt.Errorf("authDefinitions value '%s' is not supported, it must be an object or string", string(b))
+	default:
```

Comment:
If the user configured it as:
`
"auth": 123
`

There will not have error happened?

Reply:
Tried a different approach, please take a look.